pfcpiface/utils.go

```diff
@@ -99,3 +99,15 @@ func maxUint64(x, y uint64) uint64 {
 func calcBurstSizeFromRate(kbps uint64, ms uint64) uint64 {
 	return uint64((float64(kbps) * 1000 / 8) * (float64(ms) / 1000))
 }
+
+func getRemoteIP(dstIP string) net.IP {
+	conn, err := net.Dial("udp", dstIP+":"+"8806")
+	if err != nil {
+		ip := net.IPv4zero.String()
```

Comment:
Do not return zeroed IP address. Return `nil` with error instead, and then check error in node.go:63.

Reply:
Done